grunttasks/concat.js

```diff
@@ -7,6 +7,10 @@ module.exports = function (grunt) {
     requirejs: {
       dest: '<%= yeoman.tmp %>/scripts/main.js',
       src: ['app/bower_components/requirejs/require.js', '<%= yeoman.tmp %>/scripts/main.js']
+    },
+    easteregg: {
```

Comment:
I suggested doing it this way, otherwise we have to determine which environment the user is currently operating in. This can be done in a few ways, I thought of passing in configuration and checking the document.location.href, but neither of these seemed great. If we want it to run in prod only, we can just add it to prod only.